flowmachine/flowmachine/features/subscriber/entropy.py

```diff
@@ -30,12 +30,12 @@ def _make_query(self):
         GROUP BY subscriber
         """
 
-    @property
+    @abstractproperty
```

Comment:
Minor thing, but `abstractproperty` is deprecated since Python 3.3. They recommend to use
```
    @property
    @abstractmethod
```
instead (see https://docs.python.org/3/library/abc.html#abc.abstractproperty)